### PR TITLE
Fixes #812: Added privacy page

### LINKED ISSUE #812: Adding "PRIVACY" Page and linking it in the Footer
**Current behavior:**
There is no page for privacy policy in the footer.

**Expected behavior:**
There should be a link to the "PRIVACY" page in the footer on the right hand side.

**Would you like to work on this issue?**
Yes.

### PR DESCRIPTION
**Pull request by marvin08**
#### Changes proposed in this pull request:
I have added the privacy policy page of susper and linked it to the footer.

screenshot from 2017-11-15 22 16 27

screenshot from 2017-11-15 22 20 30

Fixes #812.

### CODE REVIEW

**Comment by marvin08** [comment]
Please review my PR @harshit98 @Orbiter @mariobehling

**Comment by harshit98** [comment]
@marvin08 Please check why codacy and travis is failing.

**Comment by marvin08** [comment]
Should I close this PR and open a new one?

**Comment by marvin08** [comment]
@harshit98 The checks have passed now.

File: src/app/privacy/privacy.component.html [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    <h2>Contact Us</h2> 
+            
+            <p>For information about how to contact susper, please visit our contact page.
+            </p>
+
+    </div>
+</div>
+<footer>
+    <app-footer-navbar></app-footer-navbar>
+</footer>
```
Comment (harshit98) [inline src/app/privacy/privacy.component.html]: Leave 1 line after `</footer>`

**Comment by harshit98** [comment]
@marvin08 Kindly do the suggested changes and squash your commits. The PR looks good to me! 💯

**Comment by marvin08** [comment]
@harshit98 I am closing this PR and opening a new one. Something went wrong.

File: src/app/privacy/privacy.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    const privacy = new PrivacyComponent();
+    expect(privacy).toBeTruthy();
+  });
+
+  it('should create a FooterNavbar Component', () => {
+    const footerNavbar = new FooterNavbarComponent();
+    expect(footerNavbar).toBeTruthy();
+  });
+
+});
```
Comment (aashutoshrathi) [inline src/app/privacy/privacy.component.spec.ts]: Add newline at end here too

File: src/app/privacy/privacy.component.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+	styleUrls: ['./privacy.component.css']
+})
+
+export class PrivacyComponent implements OnInit {
+	
+	constructor() { }
+
+	ngOnInit() {
+	}
+}
```
Comment (aashutoshrathi) [inline src/app/privacy/privacy.component.ts]: Add newline at end

**Pull request closed without merging**